### PR TITLE
Add "Add Paused" option for NZBget Downloader

### PR DESCRIPTION
**Pull request by vertigo235**
#### Database Migration
NO

#### Description
Adds option to add nzb to NZBGet in paused state. 

#### Todos
- [x] Tests
- [ ] Documentation

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/Download/Clients/Nzbget/Nzbget.cs [hunk 1]
```diff
@@ -44,6 +47,24 @@ protected override string AddFromNzbFile(RemoteEpisode remoteEpisode, string fil
             return response;
         }
 
+        protected override string AddFromNzbFile(RemoteMovie remoteMovie, string filename, byte[] fileContents)
```
Comment (markus101) [inline src/NzbDrone.Core/Download/Clients/Nzbget/Nzbget.cs]: Copy pasta from Radarr?

File: src/NzbDrone.Core/Download/Clients/Nzbget/Nzbget.cs [hunk 2]
```diff
@@ -44,6 +47,24 @@ protected override string AddFromNzbFile(RemoteEpisode remoteEpisode, string fil
...
+        {
+            var category = Settings.TvCategory; // TODO: Update this to MovieCategory?
```
Comment (markus101) [inline src/NzbDrone.Core/Download/Clients/Nzbget/Nzbget.cs]: Same thing here.

File: src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs [hunk 3]
```diff
@@ -55,6 +55,9 @@ public NzbgetSettings()
         [FieldDefinition(7, Label = "Use SSL", Type = FieldType.Checkbox)]
         public bool UseSsl { get; set; }
 
+        [FieldDefinition(8, Label = "Add Paused", Type = FieldType.Checkbox)]
```
Comment (markus101) [inline src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs]: Looks like this requires a specific NZBGet version, if so `HelpText` should be added to indicate that.

**Comment by vertigo235** [comment]
My mistake I did write this for sonarr and then I cherry picked it into sonarr for some reason those line dhanges didn't pop out at me. 

I'll clean it up and add the help text.

**Comment by markus101** [comment]
Thanks!

**Pull request merged**